### PR TITLE
Renamings and rewordings OFS

### PR DESCRIPTION
**Pull request by fredrik-bakke**
### Summary
- Implement a new naming scheme for files about local and separated objects at localizations/subuniverses/maps, as discussed with Egbert privately.
- Improve particular wordings in the OFS namespace.
- Add a partial converse to the lemma `is-trunc-Σ`, computes the fibers of `map-Σ` and add some coherences between composition and `map-Σ`.
- Add some basic lemmas about local maps.

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
I'm trying out a new naming scheme to specify what structure an "object" is local at.

**Comment by fredrik-bakke** [comment]
I'm still unsure about the new naming scheme I'm proposing.

For instance, it will lead to awkward phrasings such as
- `morphisms-of-arrows-cartesian-at-reflective-subuniverses`/`morphisms-of-arrows-cartesian-at-maps`
- `maps-equivalences-at-reflective-subuniverses`/`maps-equivalences-at-maps`
- `maps-connected-at-reflective-subuniverses`/`maps-connected-at-maps`

Perhaps better phrasings are
- `cartesian-morphisms-of-arrows-at-reflective-subuniverses`/`cartesian-morphisms-of-arrows-at-maps`
- `equivalences-at-reflective-subuniverses`/`equivalences-at-maps` (very weird)
- `connected-maps-at-reflective-subuniverses`/`connected-maps-at-maps` (also weird)
- `local-types-at-maps`

File: src/foundation/global-subuniverses.lagda.md [hunk 1]
```diff
@@ -126,14 +126,14 @@ module _
   {l : Level} {X Y : UU l}
   where
 
-  is-in-global-subuniverse-homogenous-equiv :
+  is-in-global-subuniverse-equiv-Level :
```
Comment (EgbertRijke) [inline src/foundation/global-subuniverses.lagda.md:129]: What would be wrong with the name `is-in-global-subuniverse-equiv`?

File: src/foundation/precomposition-dependent-functions.lagda.md [hunk 2]
```diff
@@ -120,3 +123,37 @@ is-trunc-map-succ-precomp-Π {k = k} {f = f} {C = C} H =
         ( funext (g ∘ f) (h ∘ f))
         ( H g h))
 ```
+
+### The dependent precomposition map at a dependent pair type
+
+Given a map `f : X → Y` and a family `B : (y : Y) → A y → 𝒰` we have an
+equivalence of maps
```
Comment (EgbertRijke) [inline src/foundation/precomposition-dependent-functions.lagda.md:130]: If you phrase it like this, should "equivalence of maps" be hyperlinked, and don't you think the reader will now expect a construction that gives you an equivalence of maps? Only a coherence for a commuting square is constructed.

If you rephrase this to "...we have a commuting square" you don't raise the same questions.

File: src/foundation/precomposition-functions.lagda.md [hunk 3]
```diff
@@ -213,3 +214,35 @@ module _
       ( precomp-Π f (eq-value g h))
       ( compute-htpy-eq-ap-precomp)
 ```
+
+### The precomposition map at a dependent pair type
+
+Given a map `f : X → Y` and a family `B : A → 𝒰` we have an equivalence of maps
```
Comment (EgbertRijke) [inline src/foundation/precomposition-functions.lagda.md:220]: "....we have a commuting square of maps"

(or of course feel free to actually construct an equivalence of maps)

File: src/orthogonal-factorization-systems/localizations-at-subuniverses.lagda.md [hunk 4]
```diff
@@ -22,8 +22,8 @@ open import orthogonal-factorization-systems.local-types
 Let `P` be a [subuniverse](foundation.subuniverses.md). Given a type `X`, its
 **localization** at `P`, or **`P`-localization**, is a type `Y` in `P` and a map
 `η : X → Y` such that every type in `P` is
-`η`[-local](orthogonal-factorization-systems.local-types.md). I.e. for every `Z`
-in `P`, the [precomposition map](foundation-core.function-types.md)
+`η`[-local](orthogonal-factorization-systems.types-local-at-maps.md). I.e. for
```
Comment (EgbertRijke) [inline src/orthogonal-factorization-systems/localizations-at-subuniverses.lagda.md:25]:
```suggestion
`η`[-local](orthogonal-factorization-systems.types-local-at-maps.md). I.e., for
```

File: src/orthogonal-factorization-systems/uniquely-eliminating-modalities.lagda.md [hunk 5]
```diff
@@ -31,9 +31,10 @@ open import orthogonal-factorization-systems.modal-operators
 A **uniquely eliminating modality** is a _higher mode of logic_ defined in terms
 of a monadic
 [modal operator](orthogonal-factorization-systems.modal-operators.md) `○`
-satisfying a certain [locality](orthogonal-factorization-systems.local-types.md)
-condition. Namely, that dependent precomposition by the modal unit `unit-○` is
-an equivalence for type families over types in the image of the operator:
+satisfying a certain
+[locality](orthogonal-factorization-systems.types-local-at-maps.md) condition.
+Namely, that dependent precomposition by the modal unit `unit-○` is an
```
Comment (EgbertRijke) [inline src/orthogonal-factorization-systems/uniquely-eliminating-modalities.lagda.md:36]:
```suggestion
[locality](orthogonal-factorization-systems.types-local-at-maps.md) condition,
namely that dependent precomposition by the modal unit `unit-○` is an
```

**Review by EgbertRijke — COMMENTED** [review]
Nice pull request. Only a few minor comments.

File: src/foundation/global-subuniverses.lagda.md (hunk 1 above)
Comment (fredrik-bakke) [reply src/foundation/global-subuniverses.lagda.md:129]: I would expect `is-in-global-subuniverse-equiv` to take types `X` and `Y` that may be in different universes. However, the proof of this depends on an additional axiom, as opposed to the construction that is given here.

**Comment by fredrik-bakke** [comment]
Thanks for taking the time!

File: src/foundation/global-subuniverses.lagda.md (hunk 1 above)
Comment (fredrik-bakke) [reply src/foundation/global-subuniverses.lagda.md:129]: To be clear, global subuniverses must satisfy this additional axiom, but it is not used in this particular construction, which is why I don't think it needs a particularly neat name either.

Comment (EgbertRijke) [reply src/foundation/global-subuniverses.lagda.md:129]: Thanks for the clarification!
